src/components/form/SubscriptionForm.jsx

```diff
@@ -107,7 +110,7 @@ class SubscriptionForm extends React.Component {
             validate={Validation.validName}
           />
           {showForm[state.applicant]}
-          <Button onClick={this.submitForm}>Submit</Button>
+          <Button onClick={this.submitForm} variant="contained" color="primary" disabled={state.applicant === 'advertisers'}>Submit</Button>
```

Comment:
disable advertisers submission for the time being, we'll figure out what to do later in the year